services/TwilioProgrammableSMS/index.md

```diff
@@ -60,26 +60,26 @@ Follow this steps to get started with Twilio Programmable SMS on Bluemix:
 
    ![Configure your Twilio Credentials](images/02-twilio-credentials.png)
 
+   Next, navigate to your [Phone Number Inventory](https://www.twilio.com/console/phone-numbers/incoming).  
```

Comment:
@pkamp3 for links it's better to append `{: new_window}` at the end of the link like

```markdown
[Phone Number Inventory](https://www.twilio.com/console/phone-numbers/incoming){: new_window}.
```